index.js

```diff
@@ -63,11 +103,15 @@ Script.prototype.runInThisContext = function () {
     return eval(this.code); // maybe...
```

Comment:
Firefox : 65.0.1 
throws error

`Content Security Policy: The page’s settings blocked the loading of a resource at eval (“script-src”).`

Reply:
If you are using a CSP header that does not allow eval as script-src, compliant browsers will throw an error and not let you use eval. This is working as intended.

Reply:
Also way to gravedig an unrelated PR with an issue that isn't even related to the project directly. Please don't do that.